### DISCUSSION TITLE
What are the project's goals

### DISCUSSION THREAD

**Discussion by fwsmit** [osm-search/Nominatim#2907] [Q&A]
I've been looking at the geocoding software for OSM and I feel they're all lacking a few very important features. In nominatim's case that is:
- Search suggestions
- Sorting of search results according to current map zoom
- Sorting of search results according to location (or browser language and time zone)
- Handle typo's gracefully

These are very important for a good user experience when using the search interactively, e.g. on openstreetmap.org.
I was wondering if these features are desired for the project and if there are good reasons why they haven't been implemented yet.

**Comment by lonvia** [comment]
For information about the status of specific features, please consult the discussion in the appropriate issues. For information on supporting development, see [CONTRIBUTING.md](https://github.com/osm-search/Nominatim/blob/master/CONTRIBUTING.md) and https://nominatim.org/funding/.

You also might want to check out [Photon](https://github.com/komoot/photon/).
